src/telemetry.ts

```diff
@@ -4,6 +4,7 @@
  *-----------------------------------------------------------------------------------------------*/
 
 import { getTelemetryService, TelemetryEvent, TelemetryService } from '@redhat-developer/vscode-redhat-telemetry';
+import { getStderrString } from './tkn';
```

Comment:
It would be better if you extract  `getStderrString` function in separate module, as it using is spreading across extension, and keeping it in `tkn` may provide circular import errors in future.